var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -76,7 +76,7 @@ class NetcdfC(AutotoolsPackage):
     variant("dap", default=False, description="Enable DAP support")
     variant("jna", default=False, description="Enable JNA support")
     variant("fsync", default=False, description="Enable fsync support")
-    variant("zstd", default=False, description="Enable ZStandard compression", when="@:4.8")
+    variant("zstd", default=False, description="Enable ZStandard compression", when="@:4.9.0")
```

Comment:
the other way around with the `:` ;p it's from 4.9.0 onwards that this variant should be available

Reply:
@haampie you mean `@4.9.0:`, I guess